eval/in_memory_hf_eval.yaml

```diff
@@ -30,9 +30,9 @@ fsdp_config:
 
 icl_tasks:
 -
-  label: mmlu
-  dataset_uri: local_data/mmlu.jsonl # ADD YOUR OWN DATASET URI
-  num_fewshot: [0]
-  icl_task_type: multiple_choice
-  continuation_delimiter: 'Answer: ' # this separates questions from answers
+  label: jeopardy
```

Comment:
any particular to switch to jeopardy here?

Reply:
Not really I can revert that.